src/client/components/CommentFooter/Buttons.js

```diff
@@ -211,37 +215,39 @@ class Buttons extends React.Component {
             )}
           </a>
         </Tooltip>
-        <span
-          className={classNames('CommentFooter__count', {
-            'CommentFooter__count--clickable': upVotes.length > 0 || downVotes.length > 0,
-          })}
-          role="presentation"
-          onClick={this.handleShowReactions}
-        >
-          <Tooltip
-            title={
-              <div>
-                {downVotesPreview}
-                {downVotesMore}
-                {downVotes.length === 0 && (
-                  <FormattedMessage id="no_dislikes" defaultMessage="No dislikes" />
-                )}
-              </div>
-            }
+        {downVotes.length > 0 && (
+          <span
+            className={classNames('CommentFooter__count', {
+              'CommentFooter__count--clickable': upVotes.length > 0 || downVotes.length > 0,
+            })}
+            role="presentation"
+            onClick={this.handleShowReactions}
           >
-            <FormattedNumber value={downVotes.length} />
-            <span />
-          </Tooltip>
-        </span>
-        <span className="CommentFooter__bullet" />
-        <span className="CommentFooter__payout">
-          <Tooltip title={<PayoutDetail post={comment} />}>
-            <USDDisplay
-              value={payout.cashoutInTime ? payout.potentialPayout : payout.pastPayouts}
-            />
-            <span />
-          </Tooltip>
-        </span>
+            <Tooltip
+              title={
+                <div>
+                  {downVotesPreview}
+                  {downVotesMore}
+                  {downVotes.length === 0 && (
+                    <FormattedMessage id="no_dislikes" defaultMessage="No dislikes" />
+                  )}
+                </div>
+              }
+            >
+              <FormattedNumber value={downVotes.length} />
+              <span />
+            </Tooltip>
+          </span>
+        )}
+        {payoutValue >= 0.01 && <span className="CommentFooter__bullet" />}
+        {payoutValue >= 0.01 && (
```

Comment:
How about wrapping this one up in additional <span> to avoid double condition checking?

Reply:
Wasn't sure what the best way to deal with this was, because wrapping it up in a single conditional throws linter errors (adjacent span tags need to be wrapped in another tag). If there's no issue with breaking the linter rule, or with wrapping the two spans up in another div, then yeah, I'd definitely make that change.

Reply:
There is a new feature in React that lets you group elements without need of additional div:
https://reactjs.org/docs/fragments.html